pom.xml

```diff
@@ -43,6 +43,15 @@
         <shadeBase>com.facebook.presto.hadoop.\$internal</shadeBase>
         <dep.slf4j.version>1.7.13</dep.slf4j.version>
         <dep.hadoop.version>2.7.4</dep.hadoop.version>
+
+        <air.check.skip-license>false</air.check.skip-license>
+        <air.check.skip-checkstyle>false</air.check.skip-checkstyle>
+        <air.check.skip-dependency>true</air.check.skip-dependency>
+        <air.check.skip-duplicate-finder>true</air.check.skip-duplicate-finder>
+        <air.check.skip-modernizer>true</air.check.skip-modernizer>
+        <air.check.skip-enforcer>true</air.check.skip-enforcer>
+        <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
+        <air.check.skip-pmd>true</air.check.skip-pmd>
```

Comment:
Since, you have fix various style issues, remove the skips that are unnecessary.